src/v3/src/components/Select/Select.tsx

```diff
@@ -10,68 +10,105 @@
  */
 
-import { Box } from '@mui/material';
-import { NativeSelect } from '@okta/odyssey-react';
+import {
+  FormControl,
+  FormHelperText,
+  InputLabel,
+  Select as MuiSelect,
+  SelectChangeEvent,
+} from '@mui/material';
 import { IdxOption } from '@okta/okta-auth-js/lib/idx/types/idx-js';
 import { h } from 'preact';
-import {
-  ChangeEvent, FieldElement, UISchemaElementComponent,
-} from 'src/types';
 
-import { getMessage } from '../../../../v2/ion/i18nTransformer';
 import { useAutoFocus, useOnChange, useValue } from '../../hooks';
+import {
+  UISchemaElementComponent,
+  UISchemaElementComponentWithValidationProps,
+} from '../../types';
+import { withFormValidationState } from '../hocs';
 
-const Select: UISchemaElementComponent<{
-  uischema: FieldElement
-}> = ({ uischema }) => {
+const Select: UISchemaElementComponent<UISchemaElementComponentWithValidationProps> = ({
+  uischema,
+  setTouched,
+  error,
+  setError,
+  onValidateHandler,
+}) => {
   const value = useValue(uischema);
   const onChangeHandler = useOnChange(uischema);
-  const { label, focus } = uischema;
+  const { label, focus, required } = uischema;
   const {
+    attributes,
     inputMeta: {
-      // @ts-ignore expose type from auth-js
-      messages = {},
       name,
       options,
     },
     customOptions,
   } = uischema.options;
-  const error = messages?.value?.[0] && getMessage(messages.value[0]);
   const focusRef = useAutoFocus<HTMLSelectElement>(focus);
 
-  const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
-    onChangeHandler(e.currentTarget.value);
+  const handleChange = (e: SelectChangeEvent<string>) => {
+    setTouched?.(true);
+    const selectTarget = (
+      e?.target as SelectChangeEvent['target'] & { value: string; name: string; }
+    );
+    onChangeHandler(selectTarget.value);
+    onValidateHandler?.(setError, selectTarget.value);
   };
 
   return (
-    <Box>
-      <NativeSelect
-        error={error}
-        label={label!}
-        name={name}
-        id={name}
-        value={value}
+    <FormControl
+      required={required}
+    >
+      <InputLabel htmlFor={name}>{label}</InputLabel>
+      <MuiSelect
+        native
         onChange={handleChange}
-        ref={focusRef}
+        inputRef={focusRef}
+        value={value as string}
+        error={error !== undefined}
+        inputProps={{
+          'data-se': name,
+          'aria-describedby': error && `${name}-error`,
+          name,
+          id: name,
+          ...attributes,
+        }}
       >
         {
-          [<NativeSelect.Option
-            value=""
-            key="empty"
-          />].concat(
+          [
+            <option
+              aria-label="Select an Option"
+              value=""
+              key="empty"
+            >
+              {/* TODO: OKTA-518793 - need translation key for this string */}
+              Select an Option
```

Comment:
Does the v2 widget not have this placeholder text at all? Also if we add this, make sure we address the `aria-label` part with translations.

Reply:
> Does the v2 widget not have this placeholder text at all? Also if we add this, make sure we address the aria-label part with translations.

The v2 widget has this text but it looks to be provided (by default) by courage and there is no translation key/string. I submitted a request to create the key and will update this and the aria-label in our i18n transformer (similar to others).

Reply:
> Also curious if option can be aria-labelledby an inner element rather than using explicit aria-label?

Do you mean something like: 
```suggestion
              <option
              aria-labelledby="emptyLabel"
              value=""
              key="empty"
            >
               <span id="emptyLabel">Select an Option</span>
            </option>
```

Reply:
Yeah exactly, just wondering first if labelledby/label is needed at all (i.e. is the `option` implicitly labelled by the child text?) and if it is needed then if `labelledby` is a good way to mark it rather than embedding the same text twice.

Reply:
Yeah exactly, just wondering first if labelledby/label is needed at all (i.e. is the `option` implicitly labelled by the child text?) and if it is needed then if `labelledby` is a good way to mark it rather than embedding the same text twice.

Reply:
I was looking at MUI's example, they used the aria attribute, I realized now it was because they didnt have an actual label in the option tag (we also didnt before until I found this Select an Option default string in courage). But to answer the question, we do not need this 😓 

i removed it.